Sources/CryptoSwift/CipherType.swift

```diff
@@ -12,20 +12,20 @@
     import Darwin
 #endif
 
-public enum CipherError: ErrorType {
+public enum CipherError: ErrorProtocol {
     case Encrypt
     case Decrypt
 }
 
-public protocol CipherType {
-    func cipherEncrypt(bytes: [UInt8]) throws -> [UInt8]
-    func cipherDecrypt(bytes: [UInt8]) throws -> [UInt8]
+public protocol Cipher {
```

Comment:
I prefer CipherProtocol. I just removed Cipher (enum) and this may lead to more confusion.